Add F and Say command

tomlin7 opened the pull request:
Add `F` command to the bot

tomlin7 committed:
F command

Added F command to the bot

anf reviewed (changes requested):
Change the following details and it will be merged.

tomlin7 committed:
Added cooldown and requireroles for commands

Added cooldown and requireroles for commands - 'F' and 'Say'

tomlin7 reviewed (commented):
Added cooldown and requireroles

anf reviewed (changes requested):
There are still a few errors you can work on.

tomlin7 committed:
Fixed command description, removed cooldown

Fixed command description for F command, removed cooldown

tomlin7 reviewed (commented):
Changed description, removed cooldown

tomlin7 committed:
Uppercase S

Changed S to uppercase

tomlin7 reviewed (commented):
Changed S to uppercase

tomlin7 committed:
member role no more required for F command

removed member role requirement from F command

tomlin7 committed:
removed 'say' alias from 'Say' command

removed 'say' alias from 'Say' command, because commands are not case sensitive
